pkg/test-cert/make-config

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -ex
+
+CONFIG_FILE=$1
+# Config is 1Mb
+CONFIG_SIZE_KB=4096
```

Comment:
1Mb or 4096Kb?

Reply:
Oh right, yet another artifact with the "let's try ext4"